.travis.yml

```diff
@@ -46,4 +46,13 @@ matrix:
     install:
       - pip install -r requirements.txt
       - pip install -r dev_tools/conf/pip-list-dev-tools.txt
-    script: check/pytest --ignore=cirq/contrib
+    script: check/pytest --ignore=cirq/contrib --benchmark-skip
+
+
+    - os: linux
+    env: NAME=Performance tests
+    python: "3.7"
```

Comment:
Why run performance tests with a different python version (3.7 vs 3.6)?

Reply:
Wanted to have a variety of interpreters in travis cause users do; so it increases the chances of us catching version dependent problems. Now that you mention it, different interpreters can have different performances! I will make it consistent as you suggest.

Reply:
I see. Your argument about covering more interpreter versions makes sense to me. Either is OK.